apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/IdeaContentCard/IdeaContentCard.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { atoms } from '@devlaunchers/components/src/components';
+
+const IdeaContentCard = ({ title, content, sliceLength}) => {
+
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <atoms.Layer type="light" style={{ borderRadius: "1rem", marginBottom: "3rem", padding: "1rem 2rem 2rem" }}>
+      <atoms.Box>
+        <h3>{title}</h3>
+      </atoms.Box>
+      <atoms.Box flexDirection="column">
+        {content ? (
+          <>
+            {(sliceLength && !isExpanded) ? (
+              <>
+                {content
+                  .slice(0, sliceLength)
+                  .concat("...")
+                  .split('\n')
+                  .map((paras, key) => {
```

Comment:
Might be nice to extract this function inside of the `map` to its own function for reusability, but not necessary